src/applications/gi/sass/partials/_gi-search-page.scss

```diff
@@ -134,8 +135,8 @@
     }
 
     .info-flag {
-      color: $color-white;
-      background-color: $color-gray-light;
+      color: #323a45;
```

Comment:
There is a [defined variable](https://design.va.gov/utilities/color) that would be better: `$color-gray-dark`